src/Propulsion.EventStore/Checkpoint.fs

```diff
@@ -36,44 +37,48 @@ module Fold =
     type State = NotStarted | Running of Events.Snapshotted
 
     let initial : State = NotStarted
+
```

Comment:
initial kinda belongs with evolve and fold (dotnet-templates and equinox do this pretty consistently)